### DISCUSSION TITLE
ImageMagick Version and Release Notes

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#6029] [Help]
### ImageMagick version

7

### Operating system, version and so on

ubuntu

### Description

Team where can I see all the imagemagick version and its release notes ?

**Comment by dlemstra** [comment]
You can find all the releases here: https://github.com/ImageMagick/ImageMagick/releases. And our releases notes are the ChangeLog that can be found here: https://github.com/ImageMagick/Website/blob/main/ChangeLog.md. But it's probably also a good to add the ChangeLog.md file to the releases, will check today if I can make that happen.

**Reply by programmerarunkumar** [reply]
Thank you

**Reply by pstakuu** [reply]
Do you have the release notes for versions prior to what is on the link?
